### ISSUE TITLE
%sossave magic to make full use of sos command line power.

### ISSUE THREAD

**Issue by BoPeng** [vatlab/SOS#552]
I am not quite happy with the `%runcell` and `%run` magics. The first magic execute the current cell as a workflow and the second magic execute the entire notebook (cells with section header). It seems more logical to give a larger operation a longer name, so perhaps we should give them names `%run` and `%sosrun`.

More importantly, the notebook interface does not take full advantage of the sos command line system, namely subcommands such as `sos remove`, `sos pack`, `sos unpack` etc. We could assign additional magics but it makes more sense to use these features as 

```
%sossave myworkflow.sos
!sos run myworkflow workflow parameter...
!sos remove -s
```

Basically, `%sossave` saves the current notebook as a sos script (taking only the cells with section header) so that we can execute it with commands such as `sos run`.

**Comment by BoPeng** [comment]
The new features are documented at [notebook interface](http://vatlab.github.io/SOS/doc/documentation/Notebook_Interface.html#%sosrun-148).
